decksite/scrapers/decklist_test.py

```diff
@@ -165,3 +166,97 @@ def test_parse5():
     print(d)
     assert len(d['maindeck']) == 13
     assert len(d['sideboard']) == 1
+
+# Test that a 71 card deck includes the last 15 as sideboard
+def test_parse6():
+    s = """
+        4 Animist's Awakening
+        4 Copperhorn Scout
+        15 Forest
+        4 Harvest Season
+        4 Jaddi Offshoot
+        4 Krosan Wayfarer
+        4 Loam Dryad
+        4 Nantuko Monastery
+        4 Nest Invader
+        4 Quest for Renewal
+        4 Rofellos, Llanowar Emissary
+        2 Sky Skiff
+        4 Throne of the God-Pharaoh
+        1 Distress
+        2 Dystopia
+        1 Infest
+        4 Memoricide
+        2 Nature's Ruin
+        1 Pharika's Cure
+        4 Scrabbling Claws
+    """
+    s = textwrap.dedent(s)
+    d = decklist.parse(s)
+    print(d)
+    assert sum(d['maindeck'].values()) == 61
+    assert len(d['maindeck']) == 13
+    assert len(d['sideboard']) == 7
+
+# Test a 63 card deck + 12 sideboard
+def test_parse7():
+    s = """
+        16 Forest
+        4 Animist's Awakening
+        4 Copperhorn Scout
+        4 Harvest Season
+        4 Jaddi Offshoot
+        4 Krosan Wayfarer
+        4 Loam Dryad
+        4 Nantuko Monastery
+        4 Nest Invader
+        4 Quest for Renewal
+        4 Rofellos, Llanowar Emissary
+        4 Sky Skiff
+        4 Dystopia
+        1 Infest
+        4 Memoricide
+        2 Nature's Ruin
+        1 Pharika's Cure
+        4 Scrabbling Claws
+    """
+    s = textwrap.dedent(s)
+    d = decklist.parse(s)
+    print(d)
+    assert sum(d['maindeck'].values()) == 64
+    assert sum(d['sideboard'].values()) == 12
+    assert len(d['maindeck']) == 13
+    assert len(d['sideboard']) == 5
+
+# Test a 61 card deck + 15 sideboard with one-offs around the cut
+def test_parse7():
+    s = """
+        24 Island
+        4 Curious Homunculus
+        4 Prism Ring
+        4 Anticipate
+        4 Take Inventory
+        4 Dissolve
+        3 Void Shatter
+        3 Jace's Sanctum
+        3 Whelming Wave
+        2 Control Magic
+        2 Confirm Suspicions
+        2 Counterbore
+        1 Rise from the Tides
+        1 Cryptic Serpent
+        1 Convolute
+        1 Lone Revenant
+        1 Careful Consideration
+        1 Opportunity
+        4 Annul
+        4 Invasive Surgery
+        3 Sentinel Totem
+    """
+    s = textwrap.dedent(s)
+    d = decklist.parse(s)
+    print(d)
```

Comment:
I'm not sure why we print these things but I guess the other tests do it too!

Reply:
It seems I copied it from test_5 without noticing...